lib/slosilo/adapters/sequel_adapter.rb

```diff
@@ -50,13 +50,17 @@ def each
       end
 
       def recalculate_fingerprints
-        model.each do |m|
-          m.update fingerprint: Slosilo::Key.new(m.key).fingerprint
+        # Ensures all updates are successfull, otherwise it reverts the updates
```

Comment:
```suggestion
        # Ensures all updates are successful, otherwise it reverts the updates
```

Reply:
It might perhaps be more clear to say:
```
# Use a transaction to ensure that all fingerprints are updated together. If any update fails,
# we want to rollback all updates.
```